pages/event_submit.php

```diff
@@ -0,0 +1,101 @@
+<?php
+restrict_access();
+
+require_once "database/events.api.php";
+$event = get_event_by_id(get_route_param('id_depl', true), $_SESSION['user_id']);
+$competitions = get_competitions_by_event_id($event['did'], $_SESSION['user_id']);
+
+page("Inscription - " . $event['nom'], "event_view.css");
+?>
+<form id="eventForm" method="get">
+    <div class="page-actions">
+        <a href="/evenements/<?= $event['did'] ?>" class="secondary"><i class="fas fa-caret-left"></i> Retour</a>
+    </div>
+    <article>
+        <header class="center">
+            <div class="row">
+                <div class="col-sm-6">
+                    <?php include "components/start_icon.php" ?>
+
+                    <span><?= "Départ - " . $event['depart'] ?></span>
+                </div>
+                <div class="col-sm-6">
+                    <?php include "components/finish_icon.php" ?>
+                    <span><?= "Retour - " . $event['arrivee'] ?></span>
+                </div>
+                <div>
+                    <i class="fas fa-clock"></i>
+                    <span><?= "Date limite - " . $event['limite'] ?></span>
+                </div>
+            </div>
+
+            <fieldset>
+                <label for="entrySwitch">
+                    <b>
+                        <input type="checkbox" name="event_entry" id="entrySwitch" class="entry-switch" role="switch" onchange="displayForm()" <?= $event['present'] ? "checked" : "" ?>>
+                        <ins>Je participe <i class="fas fa-check"></i></ins>
+                        <del>Je ne participe pas <i class="fas fa-xmark"></i></del>
+                    </b>
+                </label>
+            </fieldset>
+        </header>
+        <div id="conditionalForm" <?= $event['present'] ? "" : "class='hidden'" ?>>
+
+            <fieldset class="row">
+                <label for="transportSwitch" class="col-sm-6">
+                    <input type="checkbox" name="event_transport" id="transportSwitch" role="switch" <?= $event['present'] && $event['transport'] ? "checked" : "" ?>>
+                    Transport
+                </label>
+                <label for="accomodationSwitch" class="col-sm-6">
+                    <input type="checkbox" name="event_accomodation" id="accomodationSwitch" role="switch" <?= $event['present'] && $event['heberg'] ? "checked" : "" ?>>
+                    Hébergement
+                </label>
+            </fieldset>
+
+            <fieldset>
+                <label for="entryComments">
+                    Remarques:
+                </label>
+                <textarea name="event_comments"></textarea>
+            </fieldset>
+            <h4>Courses : </h4>
+            <table role="grid">
+                <?php foreach ($competitions as $competition) : ?>
+                    <tr class="display">
+                        <td class="competition-entry"><i class="fas fa-arrow-right"></i></td>
```

Comment:
Not sure if the arrow is useful here. Without anything could be good enough in my opinion.

Reply:
Removed!